### PR TITLE
In doc: mention alternative to nginx.conf templates: include-dir.

### PR DESCRIPTION
**Pull request by fwolfst**
In the line of #1303 , mention that parts of nginx-configuration can also be put in `$APP/nginx.conf.d/` if default templates are used.

Reference to comment in other issue (does not look very nice in the documentation, can probably be removed).

### CODE REVIEW

**Comment by josegonzalez** [comment]
This isn't dokku-alt,  so this actually isn't true.

**Comment by fwolfst** [comment]
As far as I can tell from the distance (I do not use dokku-alt) the templates are the same (`include **conf.d`) and it works like a charm if I didn't mess my test up.

**Comment by michaelshobbs** [comment]
@josegonzalez thoughts on adding this to the default template?

**Comment by fwolfst** [comment]
@michaelshobbs @josegonzalez It IS part of the default templates, its just not mentioned in the docs.

File: docs/nginx.md [hunk 1]
```diff
@@ -96,6 +96,21 @@ A few tips for custom nginx templates:
 
 After your changes a `dokku deploy myapp` will regenerate the `/home/dokku/myapp/nginx.conf` file which is then used.
 
+### Customizing via configuration files included by the default templates
+
+The default nginx.conf- templates will include everything from your apps `nginx.conf.d/` subdirectory in the main `server {}` block (see above):
+
+    include $DOKKU_ROOT/$APP/nginx.conf.d/*.conf;
+
+. That means you can put additional configuration in separate files, for example to limit the uploaded body size to 50 megabyte, do
```
Comment (michaelshobbs) [inline docs/nginx.md]: I think you mean megabytes?

File: docs/nginx.md [hunk 2]
```diff
@@ -96,6 +96,21 @@ A few tips for custom nginx templates:
...
+
+    mkdir /home/dokku/myapp/nginx.conf.d/
+    echo 'client_max_body_size 50M;' > /home/dokku/myapp/nginx.conf.d/upload.conf
+    chown dokku:dokku /home/dokku/myapp/nginx.conf.d/upload.conf
+    service nginx reload
+
+(taken from https://github.com/dokku-alt/dokku-alt/issues/195#issuecomment-75092205).
```
Comment (michaelshobbs) [inline docs/nginx.md]: Not sure why we would mention a different project here. Can you remove this line?

**Comment by josegonzalez** [comment]
This can be merged once the two comments are fixed.

**Pull request merged**